sbt_common/storage/src/test/scala/uk/ac/wellcome/storage/s3/S3TypeStoreTest.scala

```diff
@@ -38,7 +38,7 @@ class S3TypeStoreTest
         whenReady(writtenToS3) { actualKey =>
           val expectedJson = JsonUtil.toJson(testObject).get
 
-          val expectedKey = s"$prefix/$expectedHash.json"
```

Comment:
Why have these expected keys changed? This change in external behaviour feels wrong to me.